.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Description of change
+
+_Please provide description of change here_.
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items_:
+
+- [ ] Does `make test` (or `vcbuild test nosign`) pass with this change?
+- [ ] Is commit message formatted according to [CONTRIBUTING.md][0]
+- [ ] Regression test (or benchmark if this is a performance-related change) is
+  included (only required for code changes)
+- [ ] Documentation change or addition is included (if this change modifies
+  existing APIs, or introduces new ones)
+
```

Comment:
I don't think so. I meant having a note at the bottom of the section stating these things are not required to open a PR and can be done afterwards / while the PR is open.